src/components/common/Carousel/Carousel.scss

```diff
@@ -0,0 +1,51 @@
+.carousel-container {
+    position: relative;
+    max-width: 960px;
+    max-height: 400px;
+    overflow: hidden;
+    border-radius: 0px 0px 8px 8px;
+    background-color: var(--window-bg);
+
+    .carousel-slide {
+        display: none;
+        user-select: none;
+
+        &.active-slide {
+            display: block;
+        }
+
+        img {
+            width: 100%;
+            height: 100%;
+            max-height: 400px;
+        }
+    }
+
+    .carousel-slide__action-buttons {
+        position: absolute;
+        bottom: 0px;
+        left: 0px;
+        padding: 10px 0;
+        color: white;
+        background-color: rgba(0, 0, 0, 0.7);
```

Comment:
please use color code

Reply:
Sure, will try.
Actually, had to use the color with less opacity so used the rgba. will get the code and replace it.

Reply:
Replaced the RGBA with the color HEX.